waiter/test/waiter/service_description_test.clj

```diff
@@ -2080,6 +2080,27 @@
                                  "run-as-user" "some-user")
                           validation-settings))))
 
+    (testing "validate-service-description-namespace-star"
+      (is (nil? (validate builder
+                          (assoc basic-service-description
+                                 "namespace" "*")
+                          validation-settings))))
+
+    (testing "validate-service-description-namespace-invalid"
+      (is (thrown? Exception #"Cannot use run-as-requestor with a specific namespace"
+                   (validate builder
+                             (assoc basic-service-description
+                                    "namespace" "some-user")
+                             validation-settings))))
+
+    (testing "validate-service-description-namespace-invalid"
+      (is (thrown? Exception #"Cannot use namespace \* with specific run-as-user"
+                   (validate builder
+                             (assoc basic-service-description
+                                    "namespace" "*"
+                                    "run-as-user" "some-user")
+                             validation-settings))))
```

Comment:
Can you verify we have unit tests for and if now, can we add the test for:
- run-as-user:* and namespace:*
- run-as-user:* and namespace:foo
- run-as-user:foo and namespace:foo
- run-as-user:bar and namespace:foo

Reply:
I'm now testing 12 cases from your table in the integration tests. Only 9 of the 12 cases are tested in this unit test because it's only testing the cases covered by the `validate` method.